internal/wrappers/features/run_cpp.sh

```diff
@@ -5,7 +5,7 @@ sh ./build_cpp_tests.sh
 
 FEATURE_FILE=$1
 
-if [ ! -z "$KUZZLE_HOST" ]; then
+if [ -z "$KUZZLE_HOST" ]; then
```

Comment:
nitpicking: this may warrant a comment explaining why we do not want to output anything if a `KUZZLE_HOST` variable is set, for future reviewers